backend/schemas/scripts/render.py

```diff
@@ -499,6 +501,46 @@ def set_open_range_and_single_cell_widths(ws, sheet):
             ws.column_dimensions[column].width = r.posn.width
 
 
+def set_row_heights(wb, ws, sheet):
+    """Set the row heights based on sheet.row_height if defined."""
+    if sheet.row_height and ws.title in ("Form", "AdditionalNotes"):
+        named_cells = set()  # Set to store all cells within named ranges
+
+        # Collecting all cells that are in named ranges
+        for named_range in wb.defined_names:
+            for dest in wb.defined_names[
+                named_range
+            ].destinations:  # Each destination is a tuple of (sheet_name, cell_range)
+                if dest[0] in ("Form", "AdditionalNotes"):
+                    for row in ws[dest[1]]:
+                        for cell in row:
+                            named_cells.add((cell.row, cell.column))
+
+        # Checking and setting the height for each cell in named ranges
+        for cell_row, cell_col in named_cells:
+            ws.row_dimensions[cell_row].height = sheet.row_height
+
+
+def activate_wraptext(wb):
+    for ws in wb.worksheets:
+        if ws.title in ("Form", "AdditionalNotes"):
+            for named_range_name in wb.defined_names:
+                activate_wraptext_for_named_range(wb, ws, named_range_name)
+
+
+def activate_wraptext_for_named_range(wb, ws, named_range_name):
+    """Activate wrapText for all cells within a named range in the worksheet."""
+    if named_range_name in wb.defined_names:
+        named_range = wb.defined_names[named_range_name]
+        if named_range.attr_text.startswith(
```

Comment:
Where are these string values, with the single quotation marks and the exclamation marks, coming from?

Reply:
These are sheets named in the workbooks. For instance, for audit findings text, it is defined [here](https://github.com/GSA-TTS/FAC/pull/2518/files#diff-a5302ef12de61ad2603bca96749d4e35a95169b68e02115ee16cac8a4ef1df3bR5) and passthrough [here](https://github.com/GSA-TTS/FAC/pull/2518/files#diff-a5302ef12de61ad2603bca96749d4e35a95169b68e02115ee16cac8a4ef1df3bR104). Similarly, for CAP, it is[ here](https://github.com/GSA-TTS/FAC/pull/2518/files#diff-a5302ef12de61ad2603bca96749d4e35a95169b68e02115ee16cac8a4ef1df3bR104)

Reply:
Thinking of a way to mitigate situations where the sheet name gets changed? We could store these specific sheet names in Sheets.libsonnet and read them from there.